recipes/mlpy/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "3.5.0" %}
+{% set sha256 = "344fa75fbf9f76af72f6a346d5309613defc4d244bac13c218e509a51d68bf6a" %}
+#source file is named download, has to be unpacked manually, update build sh accordingly when updating version
+
+package:
+  name: mlpy
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/mlpy/files/mlpy%203.5.0/mlpy-{{ version }}.tar.gz/download
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+#  skip: True # [py27]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python >=2.6
```

Comment:
If you can make this `noarch: python`, you need to use the `skip` line instead of the constraint here. (Don't ask, it's how conda likes it)

Reply:
Sorry, gcc is required, noarch not possible